client/main.lua

```diff
@@ -75,37 +75,37 @@ local function scrapVehicle()
                     SetEntityAsMissionEntity(vehicle, true, true)
                     DeleteVehicle(vehicle)
                 else
-                    QBCore.Functions.Notify(Lang:t('error.canceled'), "error")
+                   exports.qbx_core:Notify(Lang:t('error.canceled'), 'error')
                 end
 
                 isBusy = false
             else
-                QBCore.Functions.Notify(Lang:t('error.smash_own'), "error")
+               exports.qbx_core:Notify(Lang:t('error.smash_own'), 'error')
             end
         else
-            QBCore.Functions.Notify(Lang:t('error.cannot_scrap'), "error")
+           exports.qbx_core:Notify(Lang:t('error.cannot_scrap'), 'error')
         end
     else
-        QBCore.Functions.Notify(Lang:t('error.not_driver'), "error")
+       exports.qbx_core:Notify(Lang:t('error.not_driver'), 'error')
     end
 end
 
 local function createListEmail()
     if cache.vehicle then return end
     if not Config.CurrentVehicles or table.type(Config.CurrentVehicles) == 'empty' then
-        QBCore.Functions.Notify(Lang:t('error.demolish_vehicle'), "error")
+       exports.qbx_core:Notify(Lang:t('error.demolish_vehicle'), 'error')
         return
     end
 
     emailSent = true
     local vehicleList = ""
     for _, v in pairs(Config.CurrentVehicles) do
-        local vehicleInfo = QBCore.Shared.Vehicles[v]
+        local vehicleInfo = exports.qbx_core:GetVehiclesByName()[v]
```

Comment:
Rather than call this export on each iteration of the for loop, it would be better to load this once and store in a variable at the top of the file.